src/mscorlib/src/System/Attribute.cs

```diff
@@ -431,7 +431,7 @@ private static AttributeUsageAttribute InternalGetAttributeUsage(Type type)
                 return AttributeUsageAttribute.Default;
 
             throw new FormatException(
-                Environment.GetResourceString("Format_AttributeUsage", type));
+SR.Format(SR.Format_AttributeUsage, type));
```

Comment:
I see 47 cases where the `SR.Format` is aligned on the left column for some reason.